sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/subscribe/SubscribeProjectPresenter.groovy

```diff
@@ -32,7 +32,7 @@ class SubscribeProjectPresenter implements SubscribeProjectOutput {
     */
     @Override
     void subscriptionAdded(String project) {
-        String message = "Subscription to ${project} was successfull"
+        String message = "Subscription to ${project} was successfull. You will receive emails informing you of updates on project ${project}."
```

Comment:
I do not think this is possible in our notifications that is why I brought it up in the planning. We might need to think of something else in case this does not provide the necessary information or is perceived as convoluted by the users.